magma/wire_container.py

```diff
@@ -39,7 +39,11 @@ def __str__(self):
     def anon(self):
         return self._bit.anon()
 
-    def unwire(self, other):
+    def unwire(self, other=None):
+        if other is None:
+            if self._driver is None:
+                raise RuntimeError("Cannot unwire undriven value")
```

Comment:
what's the argument for raising an error here? I could see it both ways, either (a) unwiring an unwired value raises, or (b) it's valid since it's like calling clear() on an empty list.

Reply:
I don't feel strongly either way.  We could make it a warning log just like overriding a driven value?